script/build.js

```diff
@@ -59,6 +59,7 @@ const optionDefinitions = [
   { name: 'watch', type: Boolean, defaultValue: false },
   { name: 'entry', type: String, defaultValue: null },
   { name: 'host', type: String, defaultValue: 'localhost' },
+  { name: 'public', type: String, defaultValue: null },
```

Comment:
The newer version of Webpack dev server forces you to use this option now if you want to expose your local server to other devices on your network.